doc/source/conda_forge_guide.rst

```diff
@@ -4,15 +4,25 @@
 How to release conda-forge package
 ==================================
 
-conda-forge: release for the first time
----------------------------------------
+.. _create-feedstock:
+
+I am new to conda-forge. How do I create a conda package?
```

Comment:
I was inspired by the way you used "I..." in the user config doc, @sbillinge 

I realized this way, it lowers activation energy and we show empathy